### DISCUSSION TITLE
Why does flameshot full only take a screenshot of part of the screen?

### DISCUSSION THREAD

**Discussion by in-in** [flameshot-org/flameshot#3085] [Q&A]
For example, this is a screenshot of the whole screen
![23037103208](https://user-images.githubusercontent.com/8797432/216912012-d934e85a-a365-45d6-b1f7-adfac2bc0ebc.png)

And this is a screenshot after running `flameshot full`
![23037103305](https://user-images.githubusercontent.com/8797432/216912070-8d1a3557-9a39-4828-816f-af86cbc6e807.png)

Yes, I use the "Use last region" option, but I don't think it should work for `flameshot full`.
![23037093147](https://user-images.githubusercontent.com/8797432/216912123-ca7cb7bc-20f2-499f-918d-b577ced475c7.png)

Flameshot v12.1.0 ()
Compiled with Qt 5.15.3

Linux Mint 21.1 Cinnamon

**Comment by mmahmoudian** [comment]
Do you have fractional scaling or Wayland?

**Reply by in-in** [reply]
No fractional scaling or Wayland
Only one display
![23037155648](https://user-images.githubusercontent.com/8797432/216977169-68d46de6-7fc1-422c-b040-ddc0ddb2a70b.png)
![23037155621](https://user-images.githubusercontent.com/8797432/216977173-16f8db4f-c16b-4624-b37d-1b6a49638791.png)

**Reply by mmahmoudian** [reply]
@in-in Thank you for being responsive and bringing up the issue. I can confirm that this indeed is a bug. Would you please file [a bug report](https://github.com/flameshot-org/flameshot/issues/new?assignees=&labels=Unconfirmed+Bug&template=bug-report.yaml). I need to assign high priority to this.

**Reply by in-in** [reply]
https://github.com/flameshot-org/flameshot/issues/3086
